### PR TITLE
chore(tiny_ttf): fix docstring on each API

### PR DESCRIPTION
**Pull request by W-Mai**
### Description of the feature or fix

A clear and concise description of what the bug or new feature is.

### Notes
- Update the [Documentation](https://github.com/lvgl/lvgl/tree/master/docs) if needed.
- Add [Examples](https://github.com/lvgl/lvgl/tree/master/examples) if relevant.
- Add [Tests](https://github.com/lvgl/lvgl/blob/master/tests/README.md) if applicable.
- If you added new options to `lv_conf_template.h` run [lv_conf_internal_gen.py](https://github.com/lvgl/lvgl/blob/master/scripts/lv_conf_internal_gen.py) and update [Kconfig](https://github.com/lvgl/lvgl/blob/master/Kconfig).
- Run `scripts/code-format.py` ([astyle](http://astyle.sourceforge.net/install.html) version [v3.4.12](https://github.com/szepeviktor/astyle/releases/tag/v3.4.12) needs to be installed) and follow the [Code Conventions](https://docs.lvgl.io/master/CODING_STYLE.html).
- Mark the Pull request as [Draft](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/changing-the-stage-of-a-pull-request) while you are working on the first version, and mark is as _Ready_ when it's ready for review.
- When changes were requested, [re-request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) to notify the maintainers.
- Help us to review this Pull Request! Anyone can [approve or request changes](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/approving-a-pull-request-with-required-reviews).

### CODE REVIEW

**Review by liamHowatt — DISMISSED** [review]
:100: thanks!

**Review by XuNeo — DISMISSED** [review]

File: src/libs/tiny_ttf/lv_tiny_ttf.h [hunk 1]
```diff
@@ -29,26 +29,60 @@ extern "C" {
  * GLOBAL PROTOTYPES
  **********************/
 
-#if LV_TINY_TTF_FILE_SUPPORT !=0
-/* create a font from the specified file or path with the specified line height.*/
+#if LV_TINY_TTF_FILE_SUPPORT != 0
+/**
+ * create a font from the specified file or path with the specified line height.
```
Comment (PGNetHun) [inline src/libs/tiny_ttf/lv_tiny_ttf.h]: Start function title with capital letter.
```suggestion
 * Create ....
```

File: src/libs/tiny_ttf/lv_tiny_ttf.h [hunk 2]
```diff
@@ -29,26 +29,60 @@ extern "C" {
...
+ * @param path        the path or file name of the font
+ * @param font_size   the font size in pixel
+ * @return a font object
+ */
 lv_font_t * lv_tiny_ttf_create_file(const char * path, int32_t font_size);
 
-/* create a font from the specified file or path with the specified line height with the specified cache size.*/
+/**
+ * create a font from the specified file or path with the specified line height with the specified cache size.
```
Comment (PGNetHun) [inline src/libs/tiny_ttf/lv_tiny_ttf.h]: Start function title with capital letter.

File: src/libs/tiny_ttf/lv_tiny_ttf.h [hunk 3]
```diff
@@ -29,26 +29,60 @@ extern "C" {
...
+ * @param cache_size  the cache size in count
+ * @return a font object
+ */
 lv_font_t * lv_tiny_ttf_create_file_ex(const char * path, int32_t font_size, lv_font_kerning_t kerning,
                                        size_t cache_size);
 #endif
 
-/* create a font from the specified data pointer with the specified line height.*/
+/**
+ * create a font from the specified data pointer with the specified line height.
```
Comment (PGNetHun) [inline src/libs/tiny_ttf/lv_tiny_ttf.h]: Start function title with capital letter.

File: src/libs/tiny_ttf/lv_tiny_ttf.h [hunk 4]
```diff
@@ -29,26 +29,60 @@ extern "C" {
...
+ * @param data        the data pointer
+ * @param data_size   the data size
+ * @param font_size   the font size in pixel
+ * @return a font object
+ */
 lv_font_t * lv_tiny_ttf_create_data(const void * data, size_t data_size, int32_t font_size);
 
-/* create a font from the specified data pointer with the specified line height and the specified cache size.*/
+/**
+ * create a font from the specified data pointer with the specified line height and the specified cache size.
```
Comment (PGNetHun) [inline src/libs/tiny_ttf/lv_tiny_ttf.h]: Start function title with capital letter.

File: src/libs/tiny_ttf/lv_tiny_ttf.h [hunk 5]
```diff
@@ -29,26 +29,60 @@ extern "C" {
...
+ * set the size of the font to a new font_size
+ * @note the font bitmap cache and glyph cache will be flushed.
+ * @param font        the font object
+ * @param font_size   the font size in pixel
+ */
 void lv_tiny_ttf_set_size(lv_font_t * font, int32_t font_size);
 
-/* destroy a font previously created with lv_tiny_ttf_create_xxxx()*/
+/**
+ * destroy a font previously created with lv_tiny_ttf_create_xxxx()
```
Comment (PGNetHun) [inline src/libs/tiny_ttf/lv_tiny_ttf.h]: Start function title with capital letter.

File: src/libs/tiny_ttf/lv_tiny_ttf.h [hunk 6]
```diff
@@ -29,26 +29,60 @@ extern "C" {
...
+ * @param kerning     kerning value in pixel
+ * @param cache_size  the cache size in count
+ * @return
+ */
 lv_font_t * lv_tiny_ttf_create_data_ex(const void * data, size_t data_size, int32_t font_size,
                                        lv_font_kerning_t kerning, size_t cache_size);
 
-/* set the size of the font to a new font_size*/
+/**
+ * set the size of the font to a new font_size
```
Comment (PGNetHun) [inline src/libs/tiny_ttf/lv_tiny_ttf.h]: Start function title with capital letter.

**Review by PGNetHun — CHANGES_REQUESTED** [review]

File: src/libs/tiny_ttf/lv_tiny_ttf.h (hunk 2 above)
Comment (W-Mai) [reply src/libs/tiny_ttf/lv_tiny_ttf.h]: Nice

File: src/libs/tiny_ttf/lv_tiny_ttf.h (hunk 4 above)
Comment (W-Mai) [reply src/libs/tiny_ttf/lv_tiny_ttf.h]: Nice

File: src/libs/tiny_ttf/lv_tiny_ttf.h (hunk 5 above)
Comment (W-Mai) [reply src/libs/tiny_ttf/lv_tiny_ttf.h]: Nice

File: src/libs/tiny_ttf/lv_tiny_ttf.h (hunk 6 above)
Comment (W-Mai) [reply src/libs/tiny_ttf/lv_tiny_ttf.h]: Nice

**Review by XuNeo — APPROVED** [review]

**Review by kisvegabor — APPROVED** [review]

**Review by FASTSHIFT — APPROVED** [review]

**Pull request merged**